jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.java

```diff
@@ -105,6 +105,10 @@ public void apply() throws ConfigurationException {
     protected void apply(SamExecutableDetector detector) throws ConfigurationException {
         SamSettings samSettings = SamSettings.getInstance();
 
+        boolean isSelected = showAllHandlerGutterIcons.isSelected();
+        this.project.getMessageBus().syncPublisher(
+            LambdaSettingsChangeListener.Companion.getTOPIC()).samShowAllHandlerGutterIconsSettingsChange(isSelected);
```

Comment:
use @JvmStatic annotation for getTopic
or create a service LambdaSettingsChangeManager.getInstance().showGutter(isSelected)